packages/stencil/package.json

```
@@ -0,0 +1,32 @@
+{
+  "name": "@nl-design-system-community/stencil",
+  "version": "0.0.1",
+  "type": "module",
+  "private": true,
+  "description": "Stencil components and helpers extracted from the website",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/nl-design-system/theme-wizard.git",
+    "directory": "packages/stencil"
+  },
+  "files": [
+    "dist/",
+    "loader/"
+  ],
+  "scripts": {
+    "build": "pnpm run '/^build:.*/'",
+    "build:stencil": "stencil build",
+    "clean": "pnpm run '/^clean:.*/'",
+    "clean:stencil": "rimraf dist/ loader/ www/",
+    "lint-build": "pnpm run '/^lint:.*/'",
+    "lint-build:ts": "tsc --noEmit --project tsconfig.json",
+    "start": "stencil build --dev --watch --serve",
```

`start` gebruiken we voor production builds, `dev` watchers enzo

```suggestion
    "dev": "stencil build --dev --watch --serve",
```